tuning_configs/dna_tuning_config.json

```diff
@@ -1,17 +1,17 @@
 {
     "population_size": 2,
-    "n_generations": 5,
+    "n_generations": 2,
     "attributes": {
```

Comment:
Some of these parameters are pretty small - only two generations?  And only a max of 32 for hidden layer size?  Is this just for the "testing" or is this for actual training?

Reply:
Yes, just for testing. When we actually tune, we can give it a bigger range.